README.md

```diff
@@ -13,6 +13,7 @@ files) from the respective projects:
 * [embedded-sht](https://github.com/Sensirion/embedded-sht/releases)
 * [embedded-sps](https://github.com/Sensirion/embedded-sps/releases)
 * [embedded-sts](https://github.com/Sensirion/embedded-sts/releases)
+* [embedded-sfm](https://github.com/Sensirion/embedded-sfm/releases)
```

Comment:
not sorted alphabetically anymore

Reply:
Hehe, didn't realize they were. Fixed :slightly_smiling_face: